### PR TITLE
feat(HMS-1907):  add dev script for containerizing provisioning env

### PR DESCRIPTION
**Pull request by amirfefer**
is needed for https://github.com/RHEnVision/provisioning-backend/pull/539

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
Could we add it to a different directory?
Our pipelines currently build container from this code base (we are not using it yet, but it's getting built).
The pipeline allows for overriding the default build container and this would override it, I suspect that's the reason for pipeline failure.

File: Dockerfile.dev [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM docker.io/node:16
```
Comment (ezr-ondrej) [inline Dockerfile.dev:1]: Docker repositories are bit restrictive at times, would it be possible to use `registry.access.redhat.com/ubi8/nodejs-16`  for this?

I've tried to figure out what https://github.com/RedHatInsights/insights-frontend-builder-common/blob/be2fa6ca7dd7a84f8efce9cdd93febf8e541a5ce/src/frontend-build.sh#L145 does, but I failed to run it localy :/

Comment (lzap) [reply Dockerfile.dev:1]: I also think we should stick with Red Hat certified containers. All of our dependencies should work as they also use them.

**Comment by lzap** [comment]
This is missing documetnation, if we want to take this route this must be fully documented ideally with a link from the main README.

**Comment by lzap** [comment]
WFM TY

**Comment by lzap** [comment]
This can be moved to https://github.com/lzap/provisioning-compose too

**Comment by amirfefer** [comment]
@lzap, the Dockerfile was moved to [provisioning-compose](https://github.com/lzap/provisioning-compose), but the `dev` script is needed here in the `package.json`, it also can be used outside the container.

**Review by ezr-ondrej — APPROVED** [review]
Not at all I did not noticed we moved the compose parts already either, sorry! :(

This looks great, no comments :+1:

**Pull request merged**